Model/Reclaim.php

```diff
@@ -47,10 +75,114 @@ public function __construct(
      * @api
      * @return string
      */
-    public function reclaim(){
+    public function reclaim()
+    {
         return $this->_klaviyoHelper->getVersion();
     }
 
+    /**
+     * Returns the Klaviyo log file
+     * 
+     * @api
+     * @return string
+     */
+    public function getLog()
+    {
+        $log = file($this->_dir->getPath('log') . '/klaviyo.log');
```

Comment:
are we able to just use the `class Handler extends \Magento\Framework\Logger\Handler\Base
` to get this? seems like it's the same thing?

Reply:
nvm